scripts/deploy-gh-pages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+mkdir tmp
+cd tmp
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy, just build to verify
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o -z "$TRAVIS_TAG" ]; then
```

Comment:
I'd split this `if` statement and add messages why we are skipping deploy, like `echo 'Skipping deploy because $TRAVIS_TAG is missing'`
I don't know where you define these environmental variables so if my comment is pointless please ignore.